data/base/endpoint/index/sub-controller.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Elementor\Data\Base\Endpoint\Index;
+
+use Elementor\Data\Base\Endpoint\Index;
+
+class SubController extends Index {
```

Comment:
SubEndpoint?

Reply:
SubEndpoint is only used when its parent is Endpoint in this case the parent of the endpoint is sub-controller.